images/federation-redirect/app.py

```diff
@@ -126,6 +126,16 @@ def set_default_headers(self):
         self.redirect(url)
 
 
+class ActiveHostsHandler(RequestHandler):
+    """Serve information about active hosts"""
+    def initialize(self, active_hosts):
+        self.active_hosts = active_hosts
+
+    async def get(self):
+        self.set_header("Content-type", "application/json")
+        self.write(json.dumps({"active_hosts": self.active_hosts}))
```

Comment:
```suggestion
        self.write({"active_hosts": self.active_hosts})
```
I learnt we can write dictionaries directly to the response and tornado takes care of headers and the like.